### DISCUSSION TITLE
[Football] Get player bye week?

### DISCUSSION THREAD

**Discussion by Kjeldgaard** [cwendt94/espn-api#380] [ESPN API] [answered]
Hi,

With bye weeks just around the corner, has anybody figured out how to determine if a player is on bye?

**Comment by cwendt94** [comment, marked as answer]
I can add a field to the box_player class for bye week. To figure out if a player is on bye, you also have to get the pro schedule of the team the player is on a check if that team is on a bye. So right now I can only add it to the box_score functionality because that currently is the only place I am getting pro schedule data.

**Reply by cwendt94** [reply]
added to package v0.25.0, field name in box_player class is `on_bye_week `

**Reply by Kjeldgaard** [reply]
You work fast! Thanks a lot.

**Reply by Kjeldgaard** [reply]
I actually ended up creating a bye week map, inspired by this https://gist.github.com/nntrn/ee26cb2a0716de0947a0a4e9a157bc1c?permalink_comment_id=4296608#gistcomment-4296608
```
res = requests.get("https://fantasy.espn.com/apis/v3/games/ffl/seasons/2022?view=proTeamSchedules_wl")
teams = res.json().get("settings").get("proTeams")
bye_week_map = {team["abbrev"].upper(): team["byeWeek"] for team in teams}
bye_week_map.update({"OAK": bye_week_map.get("LV")})
```
For some reason Devante Adams (and maybe others) plays for Oakland and not Las Vegas...
